### PR TITLE
Only remove payment type if already present.

### PR DESCRIPTION
**Pull request by remybach**
## Feature Description

Just ran into this error on `next-subscribe`.

### CODE REVIEW

File: utils/payment-type.js [hunk 1]
```diff
@@ -37,8 +37,10 @@ class PaymentType {
 		const input = this.$paymentType.querySelector(`#${type}`);
 		const label = this.$paymentType.querySelector(`label[for=${type}]`);
 
-		input.remove();
-		label.remove();
+		if (input) {
```
Comment (liamkeaton) [inline utils/payment-type.js]: Shouldn't happen by `label` could still go 💥 here

Comment (liamkeaton) [reply utils/payment-type.js]: Also worth putting a test in for this

Comment (remybach) [reply utils/payment-type.js]: Yeah, I was leaning on the fact that the way the payment types are added means that the input and label will _always_ be present together.

File: tests/utils/payment-type.spec.js [hunk 2]
```diff
@@ -89,6 +89,11 @@ describe('PaymentType', () => {
 				paymentType.hide(PaymentType.PAYPAL);
 				expect(elementStub.querySelector.calledWithMatch(PaymentType.PAYPAL)).to.be.true;
 			});
+
+			it('should fail silently if the element doesn\'t exist', () => {
+				elementStub.querySelector.returns(null);
+				expect(paymentType.hide(PaymentType.PAYPAL)).not.to.throw;
```
Comment (liamkeaton) [inline tests/utils/payment-type.spec.js]: Need to wrap this in a `()=>{}` otherwise the exception won't be caught in the `expect`

Comment (remybach) [reply tests/utils/payment-type.spec.js]: Just tried forcing the code to fail and it caught it correctly ¯\\\_(ツ)\_/¯

Comment (liamkeaton) [reply tests/utils/payment-type.spec.js]: It will blow up the test rather than saying `function threw and shouldn't`

**Pull request merged**